bundler/spec/bundler/ruby_dsl_spec.rb

```diff
@@ -21,11 +21,13 @@ class MockDSL
       :engine => engine,
       :engine_version => engine_version }
   end
+  let(:project_root) { Pathname.new("/path/to/project") }
+  before { allow(Bundler).to receive(:root).and_return(project_root) }
 
   let(:invoke) do
     proc do
       args = []
-      args << Array(ruby_version_arg) if ruby_version_arg
+      args << ruby_version_arg if ruby_version_arg
```

Comment:
Changed this to be slightly more accurate in tests because the arg is not always given as an array in the Gemfile.